src/components/tile.js

```diff
@@ -10,7 +10,9 @@ const Tile = ({ enabled, onClick, tileValue }) => {
 
     return (
         <div
```

Comment:
https://github.com/JedWatson/classnames can let you avoid this kind of ugly condition